src/Engine.cpp

```diff
@@ -569,10 +569,20 @@ bool Engine::on_add_statistics_data_series(
 
 void Engine::refresh_engine()
 {
-    clear_callback_log_();
-    clear_issue_info_();
     entity_clicked(backend::ID_ALL, backend::EntityKind::INVALID);
+}
+
+void Engine::clear_log()
+{
+    clear_callback_log_();
+    // In case there are expected callbacks emit signal to read them and
+    // update the models from the graphic thread
+    emit new_callback_signal();
+}
 
+void Engine::clear_issues()
+{
+    clear_issue_info_();
```

Comment:
It won't be needed to emit `new_callback_signal` as issues does not use the same method as callbacks.